core/server/pom.xml

```diff
@@ -249,6 +249,20 @@
           <!-- insert other configuration here (optional) -->
         </configuration>
       </plugin>
+      <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-surefire-plugin</artifactId>
+          <version>2.19</version>
+          <configuration>
+            <argLine>-Djava.net.preferIPv4Stack=true -XX:MaxPermSize=512M</argLine>
+            <redirectTestOutputToFile>${test.output.redirect}</redirectTestOutputToFile>
+            <!-- Alphabetical order on even hours, reverse-alphabetical on odd hours  -->
+            <runOrder>hourly</runOrder>
+            <reuseForks>false</reuseForks>
+            <forkCount>2</forkCount> 
```

Comment:
No.
________________________________________________
Tests run: 45, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 3.949 sec <<< FAILURE! - in alluxio.master.file.PermissionCheckTest
setStateFail(alluxio.master.file.PermissionCheckTest)  Time elapsed: 0.087 sec  <<< ERROR!
alluxio.exception.AccessControlException: Permission denied: user=admin, access=-w-, path=/testDir: failed at /
        at alluxio.master.file.PermissionCheckTest.createDirAndFileForTest(PermissionCheckTest.java:177)
        at alluxio.master.file.PermissionCheckTest.before(PermissionCheckTest.java:158)
__________________________________________